app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -145,6 +145,7 @@ class ScreenController(private val sessionRepo: SessionRepo) {
             // overlayFragment.navOverlayScrollView.updateOverlayForHomescreen(isOnHomeUrl(fragmentManager))
         } else {
             transaction.hide(overlayFragment)
+            transaction.show(fragmentManager.webRenderFragment())
```

Comment:
We only hide `WebRenderFragment` one place, on init.  Most everywhere else we seem to assume that it's always shown.  

I think it might be better to stop hiding it in the first place, rather than calling show here.  That would mean removing line 57 (where it's hidden on init) and 233 (where it is shown on `Transition.SHOW_BROWSER`).

I've done some brief testing and this seems to work as expected, although a QA pass over this would be appropriate.